### PR TITLE
Optimize zh_cn translations

### PR DESCRIPTION
**Pull request by otakustay**
Emojis are really hard to translate, some of them are duplicated, but I think it is a trivial issue since most people only requires the image to determine

### CODE REVIEW

File: js/languages/zh_cn.js [hunk 1]
```diff
@@ -37,234 +37,234 @@
...
+    "Decrease": "删除引用",
 
     // Quick Insert
-    "Quick Insert": "\u5feb\u63d2",
+    "Quick Insert": "快速拖入",
 
     // Spcial Characters
     "Special Characters": "特殊字符",
-    "Latin": "拉丁",
+    "Latin": "拉丁文",
```
Comment (Justineo) [inline js/languages/zh_cn.js]: 这几个感觉应该统一成“拉丁字母”、“希腊字母”和“西里尔字母”。

File: js/languages/zh_cn.js [hunk 2]
```diff
@@ -37,234 +37,234 @@
...
+    "Confused face": "困惑",
+    "Confounded face": "该死的",
+    "Kissing face": "面对接吻",
+    "Face throwing a kiss": "面对投掷一个吻",
+    "Kissing face with smiling eyes": "接吻脸，含笑的眼睛",
+    "Kissing face with closed eyes": "接吻的脸闭着眼睛",
+    "Face with stuck out tongue": "面对伸出舌头",
+    "Face with stuck out tongue and winking eye": "面对伸出舌头和眨动的眼睛",
+    "Face with stuck out tongue and tightly-closed eyes": "面对伸出舌头和紧闭的眼睛",
+    "Disappointed face": "失望",
```
Comment (Justineo) [inline js/languages/zh_cn.js]: 这个「……的脸」还是要的的吧？

File: js/languages/zh_cn.js [hunk 3]
```diff
@@ -37,234 +37,234 @@
...
+    "Smiling face with halo": "笑晕",
+    "Smiling face with horns": "微笑",
+    "Winking face": "眨眼",
+    "Smiling face with smiling eyes": "面带微笑的眼睛",
+    "Face savoring delicious food": "面对品尝美味的食物",
+    "Relieved face": "面对如释重负",
+    "Smiling face with heart-shaped eyes": "色",
+    "Smiling face with sunglasses": "墨镜",
+    "Smirking face": "面对面带笑容",
+    "Neutral face": "中性面",
```
Comment (Justineo) [inline js/languages/zh_cn.js]: 这一堆好像不太对……

File: js/languages/zh_cn.js (hunk 2 above)
Comment (otakustay) [reply js/languages/zh_cn.js]: 考虑到国产主流软件如QQ、微信等对表情的都显示都是没有“XX的脸”的说法，都是这样子

image

因此在此也保持这种说法了

File: js/languages/zh_cn.js (hunk 3 above)
Comment (otakustay) [reply js/languages/zh_cn.js]: 基本没办法对上，也不知道哪里有emoji的统一翻译可以查……

File: js/languages/zh_cn.js [hunk 4]
```diff
@@ -277,40 +277,40 @@ $.FE.LANGUAGE['zh_cn'] = {
...
-    "Increase quote level": "提高报价水平",
-    "Decrease quote level": "降低报价水平",
+    "Increase quote level": "提升引用层级",
+    "Decrease quote level": "降低引用层级",
     "Image / Video": "图像/视频",
-    "Resize larger": "调整大小更大",
-    "Resize smaller": "调整大小更小",
-    "Table": "表",
+    "Resize larger": "放大尺寸",
+    "Resize smaller": "缩小尺寸",
```
[the remark this answers is not in the document]
Comment (otakustay) [reply js/languages/zh_cn.js]: 我担心的是放大/缩小会被误解为视觉上的放大缩小，而不是实际尺寸的变化，类似transform和width的区别

File: js/languages/zh_cn.js [hunk 5]
```diff
@@ -37,236 +37,236 @@
...
+    "Drop image": "拖入图片",
+    "or click": "或点击",
+    "Manage Images": "管理图片",
+    "Loading": "加载中",
+    "Deleting": "删除中",
+    "Tags": "标签",
+    "Are you sure? Image will be deleted.": "将删除图片，是否确定？",
+    "Replace": "替换",
+    "Uploading": "上传中",
+    "Loading image": "图片中加载",
```
Comment (yenshih) [inline js/languages/zh_cn.js]: 图片加载中

File: js/languages/zh_cn.js [hunk 6]
```diff
@@ -37,236 +37,236 @@
...
+    "Display": "显示",
+    "Inline": "内联",
+    "Break Text": "断开文本",
+    "Alternate Text": "备用文本",
+    "Change Size": "修改尺寸",
+    "Width": "宽度",
+    "Height": "高度",
+    "Something went wrong. Please try again.": "发生了错误，请再试一次。",
```
Comment (yenshih) [inline js/languages/zh_cn.js]: “发生了错误”感觉有点奇怪，是不是我的错觉

File: js/languages/zh_cn.js [hunk 7]
```diff
@@ -37,236 +37,236 @@
...
-    "Insert Video": "\u63d2\u5165\u89c6\u9891",
-    "Embedded Code": "\u5d4c\u5165\u5f0f\u4ee3\u7801",
-    "Paste in a video URL": "粘贴在视频网址",
-    "Drop video": "放下视频",
-    "Your browser does not support HTML5 video.": "您的浏览器不支持html5视频。",
+    "Insert Video": "插入视频",
+    "Embedded Code": "嵌入式代码",
+    "Paste in a video URL": "粘贴视频网址",
+    "Drop video": "拖入视频",
+    "Your browser does not support HTML5 video.": "您的浏览器不支持HTML5视频。",
```
Comment (yenshih) [inline js/languages/zh_cn.js]: 英文两边要加空格吧，下同

File: js/languages/zh_cn.js [hunk 8]
```diff
@@ -37,236 +37,236 @@
...
+    "or click": "或点击",
+    "Manage Images": "管理图片",
+    "Loading": "加载中",
+    "Deleting": "删除中",
+    "Tags": "标签",
+    "Are you sure? Image will be deleted.": "将删除图片，是否确定？",
+    "Replace": "替换",
+    "Uploading": "上传中",
+    "Loading image": "图片中加载",
+    "Display": "显示",
```
Comment (yenshih) [inline js/languages/zh_cn.js]: 显示还是展示？

File: js/languages/zh_cn.js [hunk 9]
```diff
@@ -37,236 +37,236 @@
...
+    "Smiling face with open mouth": "大笑",
+    "Smiling face with open mouth and smiling eyes": "微笑",
+    "Smiling face with open mouth and cold sweat": "笑出汗",
+    "Smiling face with open mouth and tightly-closed eyes": "笑脸",
+    "Smiling face with halo": "笑晕",
+    "Smiling face with horns": "微笑",
+    "Winking face": "眨眼",
+    "Smiling face with smiling eyes": "面带微笑的眼睛",
+    "Face savoring delicious food": "面对品尝美味的食物",
+    "Relieved face": "面对如释重负",
```
Comment (yenshih) [inline js/languages/zh_cn.js]: “面对”如释重负？

File: js/languages/zh_cn.js (hunk 6 above)
Comment (Justineo) [reply js/languages/zh_cn.js]: 出错了？

File: js/languages/zh_cn.js (hunk 7 above)
Comment (Justineo) [reply js/languages/zh_cn.js]: 之前数字旁边也没加，统一不加我觉得也是可以的

File: js/languages/zh_cn.js (hunk 9 above)
Comment (Justineo) [reply js/languages/zh_cn.js]: “……face”都是“……的脸”的意思。

File: js/languages/zh_cn.js [hunk 10]
```diff
@@ -277,40 +277,40 @@ $.FE.LANGUAGE['zh_cn'] = {
...
     "Copy": "复制",
-    "Cut": "切",
-    "Paste": "糊",
+    "Cut": "剪切",
+    "Paste": "粘贴",
     "Basic Formatting": "基本格式",
-    "Increase quote level": "提高报价水平",
-    "Decrease quote level": "降低报价水平",
+    "Increase quote level": "提升引用层级",
+    "Decrease quote level": "降低引用层级",
```
Comment (Justineo) [inline js/languages/zh_cn.js]: 感觉还是增加减少好？

**Comment by andelf** [comment]
Sorry for not noticing your pr.
#2938 is the translation file of our own project. 
😆 

Many i18n terms have fixed translation. DO NOT INVENT YOUR OWN.
And, since emotions are most widely used in Chinese IM softwares, short-emotion names are better.

File: js/languages/zh_cn.js (hunk 3 above)
Comment (599316527) [reply js/languages/zh_cn.js]: macOS Character Viewer 可以看到苹果的翻译吧
image

**Comment by Justineo** [comment]
> Many i18n terms have fixed translation. DO NOT INVENT YOUR OWN.

Like what?

**Comment by stefanneculai** [comment]
Closing this in favor of https://github.com/froala/wysiwyg-editor/pull/2938.

**Comment by yenshih** [comment]
May I ask why did you choose https://github.com/froala/wysiwyg-editor/pull/2938?
This PR is earlier and has more experienced participants (from one of the greatest companies in China), which means all translations are based on fully discussion.
It seems there's no reason to close this PR.
@stefanneculai

**Comment by yenshih** [comment]
To @andelf 
I just have no idea about what you said "fixed translation".
May I ask for a reference or an example?

**Comment by stefanneculai** [comment]
Thanks for the input @yenshih. We don't have anyone speaking Chinese, therefore it is hard for us to decide which of the translations should go there. As per our evaluation, this PR had far more comments than the others and that is why we've chosen that one in favor of this. Do you think you could put together a PR with all the changes discussed herein reviews? That would help a lot.

**Comment by Justineo** [comment]
@stefanneculai F.Y.I. I didn’t left comment under #2938 because there was already an ongoing PR here…not because that one is better.

**Comment by stefanneculai** [comment]
Thanks for the clarification, @Justineo . If you guys could put together a new PR incorporating all the suggestions that would be amazing.

**Comment by yenshih** [comment]
@stefanneculai I strongly suggest that we should reopen this PR and continue the peer review process.

**Comment by stefanneculai** [comment]
I would be happy, however, it doesn't look any of the suggested changes were added in a new commit by @otakustay.

**Comment by otakustay** [comment]
@stefanneculai I have look at #2938 today, @andelf provides us elegant emoji translations, however I still see a bunch of obviously incorrect translation for other commands

Since #2938 is merged currently, this PR is in a merge conflict state, I'll try to further investigate all these translations these days and make a new PR to resolve translation issues this weekend

I'll make my best to align translations for common terms to align with Microsoft Word's Chinese version, for those absent in Microsoft Word, I will mark them with comments and open to discussion

Thanks

**Comment by stefanneculai** [comment]
@otakustay Thanks a million!!

**Comment by otakustay** [comment]
@stefanneculai I've created a patch pull request to @andelf 's PR at https://github.com/andelf/wysiwyg-editor/pull/1 , expecting to discuss and merge our translations there

**Comment by stefanneculai** [comment]
Thanks @otakustay. Would you consider #2938 ready to be merged now?

**Comment by otakustay** [comment]
@stefanneculai Yes, time to merge that, thanks

**Pull request closed without merging**